repository/project.go

```diff
@@ -13,22 +14,26 @@ import (
 )
 
 type ProjectRepository struct {
-	client *supabase.Client
-	entity string
+	client            *supabase.Client
+	projectEntity     string
```

Comment:
Interesting pattern. But doesn't feel natural, doesn't feel right.
For example because:
1) Name - `entity` gives a feeling that it is a `struct`, but it is only a `string`.
2) If it's a constant, then well, wouldn't it be nicer to simply define `constant`'s for it?

Reply:
You are right, good point. I would go for constant and name it for example `projectDBTable`